blocks/library/paragraph/editor.scss

```diff
@@ -1,4 +1,4 @@
-.editor-block-list__block:not( .is-multi-selected ) .wp-block-paragraph {
+.editor-block-list__block:not( .is-multi-selected ) .wp-block-paragraph:not( .has-background ) {
```

Comment:
Also mentioned at https://github.com/WordPress/gutenberg/pull/5658#discussion_r176419793, but why do we assign a white background at all? What about when themes override default editor styles and provide their own background colors?

Reply:
The reason why we set a white background is related to the contrast verification. If we the background was never set get computed background style returns back (if I remember correctly). So adding a background fixes the problem. If themes change the background it also works well because as long as a background is set we are able to retrieve it. The problem happens when no background is set and as we get back we cannot differentiate and assume is white because in fact it may be set to black. I will try to research this in more detail maybe there is a way to detect when the background is not set and in that case assume white.

Reply:
Can we _set the white background_ on the fly, when the contrast checker is working?